packages/cdr/docker/jsreport/report-1e/content.handlebars

```diff
@@ -0,0 +1,194 @@
+      series1.dataFields.valueX = "malesPercent";
+      series1.dataFields.categoryY = "daysGroup";
+      series1.stacked = true
+      series1.name = "Male"
+
+      var series2 = chart.series.push(new am4charts.ColumnSeries());
+      series2.dataFields.valueX = "femalesPercent";
+      series2.dataFields.categoryY = "daysGroup";
+      series2.stacked = true
+      series2.name = "Female"
+
+      // chart 2
+      var chart = am4core.create("chartdiv2", am4charts.XYChart);
+      chart.data = data.rows
+      chart.legend = new am4charts.Legend();
+      chart.events.on('ready', () => {
+        chartLoaded('chart3')
+      })
+
+      var title = chart.titles.create();
+      title.text = "New HIV diagnosis by Sex and Time to ART";
+      title.fontSize = 25;
+      title.marginBottom = 30;
+
+      chart.bottomAxesContainer.layout = "horizontal";
+
+      var valueAxis = chart.yAxes.push(new am4charts.ValueAxis());
+      
+
+      var categoryAxis1 = chart.xAxes.push(new am4charts.CategoryAxis());
+      categoryAxis1.dataFields.category = "daysGroup";
+      categoryAxis1.renderer.labels.template.rotation = -90;
+      categoryAxis1.renderer.labels.template.dx = -15;
+      categoryAxis1.renderer.minGridDistance = 20;
+
+      var series1 = chart.series.push(new am4charts.ColumnSeries());
+      series1.dataFields.categoryX = "daysGroup";
+      series1.dataFields.valueY = "males";
+      series1.xAxis = categoryAxis1
+      series1.name = "Male"
+
+      var categoryAxis2 = chart.xAxes.push(new am4charts.CategoryAxis());
+      categoryAxis2.dataFields.category = "daysGroup";
+      categoryAxis2.renderer.labels.template.rotation = -90;
+      categoryAxis2.renderer.labels.template.dx = -15;
+      categoryAxis2.renderer.minGridDistance = 20;
+
+      var series2 = chart.series.push(new am4charts.ColumnSeries());
+      series2.dataFields.categoryX = "daysGroup";
+      series2.dataFields.valueY = "females";
+      series2.xAxis = categoryAxis2
+      series2.name = "Female"
+
+    })
+  }
+
+  const scriptsToLoad = [
+    'https://cdn.amcharts.com/lib/4/core.js',
+    'https://cdn.amcharts.com/lib/4/charts.js',
+  ]
+
+  function createScriptElement() {
+    // gets the first script in the list
+    const script = scriptsToLoad.shift()
+    // all scripts were loaded
+    if (!script) {
+      console.log('All scripts loaded, rendering charts')
+      renderCharts()
+      return
+    }
+
+    console.log(`loading script ${script}`)
+    const js = document.createElement('script')
+    js.type = 'text/javascript'
+    js.src = script
+    js.onload = loadNextScript
+    const s = document.getElementsByTagName('script')[0]
+    s.parentNode.insertBefore(js, s)
+  }
+
+  function loadNextScript() {
+    // loads the next script
+    createScriptElement()
+  }
+  loadNextScript()
+
+  // wait for the charts to load and animations to complete
+  const waitingForCharts = ['chart1', 'chart2', 'chart3']
```

Comment:
Seems like only chart 1 and 2 are necessary.

Reply:
done

Reply:
done